src/quo2/components/dropdowns/network_dropdown/view.cljs

```diff
@@ -0,0 +1,32 @@
+(ns quo2.components.dropdowns.network-dropdown.view
+  (:require [quo2.theme :as quo.theme]
+            [quo2.components.dropdowns.network-dropdown.style :as style]
+            [react-native.core :as rn]
+            [quo2.foundations.resources :as quo.resources]
+            [quo2.components.list-items.preview-list :as preview-list]
+            [reagent.core :as reagent]))
+
+(def networks-list
+  [{:source (quo.resources/get-network :ethereum)}
+   {:source (quo.resources/get-network :optimism)}
+   {:source (quo.resources/get-network :arbitrum)}])
+
+(defn- internal-view
+  [_]
+  (let [pressed? (reagent/atom false)]
+    (fn
+      [{:keys [on-press state] :as props}]
+      [rn/pressable
+       {:style               (style/dropdown-container (merge props {:pressed? @pressed?}))
+        :accessibility-label :network-dropdown
+        :disabled            (= state :disabled)
+        :on-press            (when on-press on-press)
```

Comment:
This should be `#(when on-press on-press)`

Reply:
What is the need for `when`? Isn't just `on-press` enough?

Reply:
@OmarBasem, nil-check habit) but I agree that this check for sure will happen inside `rn/pressable`, so there is no reason to do it